aspnetcore/fundamentals/host/hosted-services/samples/2.x/BackgroundTasksSample-GenericHost/Program.cs

```diff
@@ -18,6 +18,10 @@ public static async Task Main(string[] args)
                     config.AddConsole();
                     config.AddDebug();
                 })
+                .ConfigureHostConfiguration(configHost =>
+                {
+                    configHost.AddEnvironmentVariables(prefix: "ASPNET_");
```

Comment:
For this sample let's drop the prefix. Nothing here is aspnetcore specific.

Reply:
... and we won't sweat *hosting.json* or command-line args then.

Reply:
Ok, getting rid of the prefix